{"name": "test_auth_biometric", "status": "failed", "statusDetails": {"message": "AssertionError: Expected 'Failed', but got 'None'\nassert None == 'Failed'", "trace": "self = <TestCases.test_biometric_auth.Test_Bio_Auth object at 0x000001A85A28B410>\n\n    def test_auth_biometric(self):\n        home = HomeScreen(self.driver)\n        home.goToAndroidBioMetric().clickOnPassCallBack()\n        actualMessage = home.goToAndroidBioMetric().getStatusMessage()\n    \n        expectedMessage = 'Failed'\n    \n>       assert actualMessage == expectedMessage, f\"Expected '{expectedMessage}', but got '{actualMessage}'\"\nE       AssertionError: Expected 'Failed', but got 'None'\nE       assert None == 'Failed'\n\ntest_biometric_auth.py:14: AssertionError"}, "attachments": [{"name": "log", "source": "d6dc05e1-5966-482c-9ee2-08703e091017-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "0d049d9e-d2ae-46a8-85df-0de74dfcd82a-attachment.txt", "type": "text/plain"}], "start": 1754397706169, "stop": 1754397708863, "uuid": "efe6890f-1810-458e-8deb-cdeb4b67e3ff", "historyId": "476a68ff226e5853fa3aa36fa902fed3", "testCaseId": "476a68ff226e5853fa3aa36fa902fed3", "fullName": "test_biometric_auth.Test_Bio_Auth#test_auth_biometric", "labels": [{"name": "suite", "value": "test_biometric_auth"}, {"name": "subSuite", "value": "Test_Bio_Auth"}, {"name": "host", "value": "Abhishek"}, {"name": "thread", "value": "19408-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_biometric_auth"}], "titlePath": ["test_biometric_auth.py", "Test_Bio_Auth"]}